src/pytorch/neighbors/getNeighborPairsCPU.cpp

```diff
@@ -25,6 +28,11 @@ static tuple<Tensor, Tensor, Tensor> forward(const Tensor& positions,
 
     TORCH_CHECK(cutoff.to<double>() > 0, "Expected \"cutoff\" to be positive");
 
+    if (box_vectors.size(0) != 0) {
```

Comment:
It could check here if all the requirements are satisfied.

Reply:
I added the checks in the CPU version.